test/e2e/e2e.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# standard bash error handling
+set -o errexit;
+set -o pipefail;
+set -o nounset;
+# debug commands
+set -x;
+
+# cleanup on exit (useful for running locally)
+cleanup() {
+    kind delete cluster || true
+}
+trap cleanup EXIT
+
+main() {
+    ROOT="$(git rev-parse --show-toplevel)"
+
+    # Setup
+    kind create cluster --loglevel=debug --config "${ROOT}/test/e2e/kind-cluster.yaml"
+    kind export kubeconfig
+
+    # Install
+
+    # TODO: ko fails if we try to run/source install.sh, but doesn't if we inline verbatim.
```

Comment:
I wonder if this is because the install script now sets KIND_CLUSTER_NAME to "tekton-results" if it hasn't been set elsewhere.  ( I made this change in my most recent PR to the script because otherwise running 00-setup.sh followed by 01-install.sh resulted in an error locally because I didn't have KIND_CLUSTER_NAME defined or exported ).

It looks like the setup step here is creating a cluster named the default, "kind"?  Wonder if a simple `export KIND_CLUSTER_NAME="kind"` at the top of this script would resolve the issue running `install.sh`?

Reply:
🤦  that totally did it!